config.js

```diff
@@ -1,5 +1,6 @@
 export const url = "https://develop.openbadges.education";
 export const backendUrl = "https://api.develop.openbadges.education";
 export const defaultWait = 12000;    // waits for a specified amount of time for a certain condition to be true. 
-export const implicitWait = 2000;   // waits for a specified amount of time before throwing a no such element exception. 
+// TODO: This was 2000. Why do we have this? This makes some cases incredibly slow. We shouldn't use implicit wait
+export const implicitWait = 0;   // waits for a specified amount of time before throwing a no such element exception. 
```

Comment:
Even the [docs](https://www.selenium.dev/documentation/webdriver/waits/) mention, we shouldn't mix both explicit and implicit waits, so this seems like the right thing to do.
`0` also is the default value, so I think we should not leave a TODO here and just get rid of it entirely.

@YouQam is not on the OEB project anymore, so I don't really expect we will get some input any time soon.